src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -6605,6 +6570,98 @@ public override IOperation VisitRangeOperation(IRangeOperation operation, int? a
             return new RangeOperation(operation.IsLifted, operation.IsImplicit, semanticModel: null, operation.Syntax, operation.Type, visitedLeftOperand, visitedRightOperand, operation.Method);
         }
 
+        public override IOperation VisitSwitchExpression(ISwitchExpressionOperation operation, int? captureIdForResult)
```

Comment:
This implementation looks fine to me for now. The verifier fails because there's probably a new case it needs to know about for potential long-lived locals (ie, it's verifying that regions are as compacted as they can be), but this should be enough for not breaking catastrophically breaking ide features.